packages/extension/.vscode/launch.json

```diff
@@ -18,7 +18,8 @@
         "${workspaceFolder}/dist/**/*.js"
       ],
       "env": {
-        "NODE_ENV": "development"
+        "NODE_ENV": "development",
+        "SQLTOOLS_DEBUG_PORT_LS": "6010"
```

Comment:
What's this for? I don't see it referenced anywhere?

Reply:
This is the port that the `attach` profile attaches to. I'm not sure why it wasn't present initially, as the value used by this package (`extension`) is 6010 (see a few lines below), but the default is 6011 (https://github.com/mtxr/vscode-sqltools/blob/b8938cafa59dd378d22a7ef24ecb710de51b400e/packages/extension/src/language-client/client.ts#L125)

Reply:
this is to start language server in the debugger mode and that is the port that will be used